server/common/stats/type.ts

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+export interface MAPS_STATS {
+  maps_total: number;
+  layers_filters_total: number;
+  layers_count: LAYER_STATS;
+  maps: MAP_STATS[];
+}
+
+export interface MAP_STATS {
+  id: string;
+  layers_filters_total: number;
+  layers_count: LAYER_STATS;
+}
+
+export interface LAYER_STATS {
+  total: number;
+  documents: number;
```

Comment:
Why not just use layer type as key and count as number. We don't have to update this stats when new layer is added.

Reply:
Good idea, updated